src/pages/Home.jsx

```diff
@@ -0,0 +1,25 @@
+import { Box, Flex, Heading, Text } from '@chakra-ui/react';
+import SearchBar from '../components/SearchBar';
+import imageSrc from '../../src/assets/images/background-image.png';
```

Comment:
```suggestion
import imageSrc from '../assets/images/background-image.png';
```
Tu sors du dossier `src` pour y retourner : pas besoin 🤷